routingpy/routers/google.py

```diff
@@ -378,7 +349,7 @@ def _parse_direction_json(self, response, alternatives):
 
         directions = []
         for route in response["routes"]:
-            duration, distance, geometry, departure_datetime, arrival_datetime = self._parse_legs(
+            duration, distance, geometry, departure_datetime, arrival_datetime = Google.parse_legs(
```

Comment:
Maybe it's just me, but it seems a bit odd. But I can live with it and we stay consistent with the rest of the code, which is the most important thing.